sdk/opa_test.go

```diff
@@ -225,13 +226,13 @@ func TestDecisionWithStrictBuiltinErrors(t *testing.T) {
 			"main.rego": `
 				package example
 
-erroring_function(number) = output {
-	output := number / 0
-}
+				erroring_function(number) = output {
+					output := number / 0
+				}
 
-allow {
-	erroring_function(1)
-}`,
+				allow {
+					erroring_function(1)
+				}`,
```

Comment:
Some adjustments to match rest of file, missed in https://github.com/open-policy-agent/opa/pull/5438

Reply:
😄 me too, but I just wanted to match what was already there in the rest of the file. I can unindent them all since that does seem to be more correct.

Reply:
815d5b75 updates the rego examples in the sdk tests to use this convention.